process/smartContract/process.go

```diff
@@ -1150,6 +1063,17 @@ func (sc *scProcessor) processSimpleSCR(
 	return nil
 }
 
+// [NotConcurrentSafe]
+func (sc *scProcessor) setSilentError(err error) {
+	sc.lastSilentError = err
```

Comment:
smart contract processor is stateless - why do you keep anything like this. The error set is returned in the SCR when created if smart contracgt execution failed.
There is not need for this set and get.

Reply:
Moved the "get last error" to `testUtils`, to retrieve it from the list of "intermediary transactions".